build failure with `--all-features`

repo is @ 2b45011c9bed331319837eceb2b593e84b5b9cfa

`cargo check --all-features`
````
    Checking linked-hash-map v0.5.4
    Checking regex-syntax v0.6.22
   Compiling memchr v2.3.4
    Checking thread_local v1.1.3
    Checking terminal_size v0.1.16
    Checking yaml-rust v0.4.5
   Compiling clap_derive v3.0.0-beta.2 (/home/matthias/vcs/github/clap/clap_derive)
    Checking aho-corasick v0.7.15
    Checking textwrap v0.13.2
    Checking regex v1.4.3
    Checking clap v3.0.0-beta.2 (/home/matthias/vcs/github/clap)
error[E0425]: cannot find value `o` in this scope
    --> src/parse/parser.rs:1352:80
     |
1352 |                     debug!("Parser::remove_overrides:iter:{:?} => {:?}", name, o);
     |                                                                                ^ not found in this scope

error[E0425]: cannot find value `o` in this scope
    --> src/parse/parser.rs:1367:31
     |
1367 |                         name, o
     |                               ^ not found in this scope

error: aborting due to 2 previous errors
````

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -89,6 +89,12 @@ jobs:
         with:
           command: test
           args: --target ${{ matrix.target }} --features "wrap_help yaml regex unstable"
+      - name: Check debug statement
+        uses: actions-rs/cargo@v1
+        if: matrix.features == 'all'
+        with:
+          command: check
+          args: --target ${{ matrix.target }} --features "wrap_help yaml regex unstable debug"
       - name: Test release
         uses: actions-rs/cargo@v1
         if: matrix.features == 'release'

--- a/src/parse/parser.rs
+++ b/src/parse/parser.rs
@@ -1349,7 +1349,10 @@ impl<'help, 'app> Parser<'help, 'app> {
             if let Some(overrider) = self.app.find(name) {
                 let mut override_self = false;
                 for overridee in &overrider.overrides {
-                    debug!("Parser::remove_overrides:iter:{:?} => {:?}", name, o);
+                    debug!(
+                        "Parser::remove_overrides:iter:{:?} => {:?}",
+                        name, overrider
+                    );
                     if *overridee == overrider.id {
                         override_self = true;
                     } else {
@@ -1364,7 +1367,7 @@ impl<'help, 'app> Parser<'help, 'app> {
                 {
                     debug!(
                         "Parser::remove_overrides:iter:{:?}:iter:{:?}: self override",
-                        name, o
+                        name, overrider
                     );
                     self_override.push(overrider.id.clone());
                 }

